### ISSUE TITLE
Strange results for 24k training

### ISSUE THREAD

**Issue by JBloodless** [Rikorose/DeepFilterNet#338]
Hi! I'm trying to train 24k model this time, and the results are kinda unexpected. I've already trained my own versions of 48k and 16k models, and they work fine, but 24k behaves weirdly.

0. In my configs for 48k and 16k models I set following parameters:
```
sr = 48000
fft_size = 960
hop_size = 480
nb_erb = 32
nb_df = 96 
```
for 48k

```
sr = 16000
fft_size = 480
hop_size = 240
nb_erb = 16
nb_df = 48 
```
for 16k.

So, based on these, for 24k model I assumed 
```
sr = 24000
fft_size = 720
hop_size = 360
nb_erb = 24
nb_df = 72
```

1. First weird thing happend on train. Here's the screenshots from tensorboard:
<img width="1421" alt="Снимок экрана 2023-05-25 в 15 42 26" src="https://github.com/Rikorose/DeepFilterNet/assets/8794436/6360b650-149c-4b27-b785-9983fc29821e">
<img width="1414" alt="Снимок экрана 2023-05-25 в 15 42 58" src="https://github.com/Rikorose/DeepFilterNet/assets/8794436/a32d4cd1-de07-4ff8-b79b-d1838579a025">
Top blue one is 48k model, middle green is 16k, lowest one is 24k. 24k model should be between them...

Moreover, validation loss is very weird. Here's val loss for 16k model:
<img width="1435" alt="Снимок экрана 2023-05-25 в 15 45 17" src="https://github.com/Rikorose/DeepFilterNet/assets/8794436/1e80a1d8-7c88-47d1-9bb7-ed85af12698d">
Seems legit. But 24k model is all over the place and loss is way too low...
<img width="1418" alt="Снимок экрана 2023-05-25 в 15 45 59" src="https://github.com/Rikorose/DeepFilterNet/assets/8794436/d2bef7a2-57c4-4fde-bf1f-629a356df250">

2. The results of denoise are strange. Top to bottom:
original - 48k result - 16k result - 24k result
<img width="1680" alt="Снимок экрана 2023-05-25 в 15 48 33" src="https://github.com/Rikorose/DeepFilterNet/assets/8794436/6179ec79-aa1d-47cd-84d9-ec9ee0dbc44d">
Original 48k input obviously resampled to corresponding samplerate before feeding into the model.

Datasets was preprocessed with `prepare_data.py`, for 16k model I set `--sr 16000`, for 24k model `--sr 24000`. Originally there are bunch of datasets in 48kHz AND 16kHz, as you can see, this method works fine for 48k and 16k model, but I have my fair share of doubts for 24k - is this legit method for preparing data this way? 
There's literally nothing else that's different between these 3 model, except different .hdf5 for training and different parameters in `[df] `section of` config.ini `which mentioned above, so what could be the problem here?

**Comment by JBloodless** [comment]
Update: i've tried 
```
sr = 24000
fft_size = 480
hop_size = 240
nb_erb = 16
nb_df = 96
```
which should be right for selecting bottom 5khz for DF, and it also fails. Could this be a problem with data? None of my datasets are 24khz natively, they are upsampled from 16k and downsampled from 48k with `prepare_data.py`
